config.sample.yaml

```diff
@@ -607,6 +601,18 @@ ircService:
     # Ignore users mentioned in a io.element.functional_members state event when checking admin room membership
     ignoreFunctionalMembersInAdminRooms: false
 
+  # Config for the media proxy, required to serve publically accessible URLs to authenticated Matrix media
+  mediaProxy:
+    # To generate a .jwk file:
+    # $ yarn -s ts-node src/generate-signing-key.ts > signingkey.jwk
```

Comment:
Should we make a script entry for this to shortcut it a bit.

And is `ts-node` available by default?

Reply:
how chunky is it?